CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.22.5] - 2021-10-06
+- Fix bug for generateDataTemplateTask to consume command line option correctly
```

Comment:
Would suggest updating the changelog to be a little more specific about how the command-line args have changed.